### ISSUE TITLE
Support global config

### ISSUE THREAD

**Issue by abdonrd** [mrmlnc/vscode-csscomb#33]
Support a global config for the projects haven't `.csscomb.json` file.

**Comment by mrmlnc** [comment]
Small clarification: if you has `.csscomb.json` file in `$HOME` directory is not works?

**Comment by abdonrd** [comment]
@mrmlnc no :/

**Comment by mrmlnc** [comment]
You can set filepath to config in the `csscomb.preset` option in **3.2.1** version.

**Comment by abdonrd** [comment]
@mrmlnc I set:

```
"csscomb.preset": "~/Work/.csscomb.json"
```

And I have this output:

>Error: "~/Work/.csscomb.json" is not a valid config name. Try one of the following: 'csscomb', 'yandex', 'zen'.

**Comment by mrmlnc** [comment]
Sorry. I marked this issue as a `bug` because I don't have any time at the moment.

**Comment by mrmlnc** [comment]
Please, try to use vscode-csscomb@**3.2.2**.

**Comment by abdonrd** [comment]
@mrmlnc the same problem at https://github.com/mrmlnc/vscode-csscomb/issues/33#issuecomment-257156771 😕

**Comment by mrmlnc** [comment]
😢

**Comment by mrmlnc** [comment]
Aaaaaaaand try to use 3.2.3 😄

**Comment by abdonrd** [comment]
@mrmlnc now different error:

>Error: ENOENT: no such file or directory, open '/~/Work/.csscomb.json'

Maybe the start `/`?

But my setup is:

```
"csscomb.preset": "~/Work/.csscomb.json"
```

**Comment by mrmlnc** [comment]
Sorry, I not have macOS/Linux now and tomorrow I'm going to install Linux in VirtualBox. The problem is that I need to handle three ways:

  * Windows paths
  * Unix-like paths with `~`
  * The path relative to the current directory

**Comment by mrmlnc** [comment]
Well, `vscode-csscomb@3.2.4` work correctly with this Issue on Manjaro & ElementaryOS. Please, try 3.2.4...

**Comment by abdonrd** [comment]
@mrmlnc the same problem at https://github.com/mrmlnc/vscode-csscomb/issues/33#issuecomment-260719604 😕

**Comment by mrmlnc** [comment]
👀 OS?

**Comment by abdonrd** [comment]
macOS Sierra 10.12.1

**Comment by abdonrd** [comment]
Any news? :)

**Comment by mrmlnc** [comment]
Fixed in 3.2.5. Tested on Windows 10, Manjaro, Elementary OS and macOS Sierra 10.12 Beta 8.

**Comment by abdonrd** [comment]
Thanks! 🎉
